tests/migration_test.go

```diff
@@ -52,7 +52,8 @@ import (
 	"github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
 	k8sv1 "k8s.io/api/core/v1"
-	"k8s.io/api/policy/v1beta1"
+	policyv1 "k8s.io/api/policy/v1"
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
```

Comment:
Why do we keep in tests the beta version?

Reply:
The `policyv1beta1` is used only  for the `Eviction` type in the `Pods(ns).Evict()` function. Since the purpose of the PR was to update the `poddisruptionbudget` api calls, I think it was misleading mix the things. Furthermore the `Pods(ns).Evict()` has been deprecated from 1.22 in favor of `EvictV1` or `EvictV1beta1` (just for info). What do you think? Thanks

Reply:
So you are saying we can remove this in a following PR? (and we do not do it here in order not to mix the two things?)